packages/slice-machine/lib/models/common/MockConfig.ts

```diff
@@ -100,7 +100,7 @@ export const SliceMockConfig = {
 
 export interface CustomTypeMockConfig {
```

Comment:
Do we still need this or is it a legacy type also ?

Reply:
it's still used in some places to manipulate the generated legacy mock config so for now it can remain there.

Reply:
That's why I asked in a comment above about typing the legacy mocks

Slice Legacy Mocks,
Custom type legacy Mocks,
....

I just don't like that there is multiple old legacy code a bit scattered that we might not know about.

Reply:
see previous answers.